converter/converter.py

```diff
@@ -15,8 +16,13 @@ def celsius_to_fahrenheit(temperature_in_celsius: float) -> float:
     Temp: 20 C
     20 * (9/5) + 32 = 68 F
     """
-    fahrenheit = round(((temperature_in_celsius * (9 / 5)) + 32), 1)
-    return fahrenheit
+    if temperature_in_celsius < ABSOLUTE_ZERO_CEL:
+        quit(f"Error temperature couldn't be lower than {ABSOLUTE_ZERO_CEL}")
```

Comment:
@MariaMaximova, please use exceptions and add unit tests for raising exception when passing temperature lower than absolute minimum